src/components/AppNavigation/Trashbin.vue

```diff
@@ -42,6 +42,14 @@ License along with this library. If not, see <http://www.gnu.org/licenses/>.
 					</EmptyContent>
 					<template v-else>
 						<h2>{{ t('tasks', 'Trash bin') }}</h2>
+						<ActionButton
```

Comment:
`ActionButton` needs to be wrapped inside and `Actions` component. See https://github.com/nextcloud/tasks/pull/1802/files#diff-a605b8ea178482e0a0bcab69e53d101699e820dde4ae75e617a1d993bd92c634R84-R92 for an example.